src/app/camping-vliegenbos/page.tsx

```diff
@@ -0,0 +1,74 @@
+'use client'
+
+import { Card, Grid, Heading, Image, Paragraph } from '@amsterdam/design-system-react'
+import campingVliegenbosPicture from './_assets/camping-vliegenbos-picture.jpg'
+import formatPath from '../../utils/formatPath'
+
+const cards = [
+  {
+    label: 'Over de camping',
+    href: '/camping-vliegenbos/over-de-camping',
+  },
+  {
+    label: 'Zoek en boek',
+    href: '/camping-vliegenbos/zoek-en-boek',
+  },
+  {
+    label: 'Route',
+    href: '/camping-vliegenbos/route',
+  },
+  {
+    label: 'Tenten',
+    href: '/camping-vliegenbos/tenten',
+  },
+  {
+    label: 'Campers en caravans',
+    href: '/camping-vliegenbos/campers-en-caravans',
+  },
+  {
+    label: 'Trekkershutten',
+    href: '/camping-vliegenbos/trekkershutten',
+  },
+]
+
+export default function HomePage() {
+  return (
+    <>
+      <Image className="ams-mb-xl" alt="" src={campingVliegenbosPicture.src} aspectRatio="16:5" />
```

Comment:
We can use Next Image here, similar to [what we do on the amsterdam.nl homepage](https://github.com/Amsterdam/design-system-prototypes/blob/main/src/app/amsterdam/page.tsx#L106)